### PR TITLE
Chore: Expect by test id instead of generic tag

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Be more specific in tests.

# Changes

* Change the expected article tag with the actual test id expected to be rendered.

# Tests

* Changes in IneligibleNeuronsCard.spec, VotingHistoryModal.spec, NnsAccounts.spec and NnsProposals.spec

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/lib/pages/NnsAccounts.spec.ts [hunk 1]
```diff
@@ -66,9 +65,11 @@ describe("NnsAccounts", () => {
         hardwareWallets: [],
         certified: true,
       });
-      const { container } = render(NnsAccounts, { props: { goToWallet } });
+      const { queryAllByTestId } = render(NnsAccounts, {
+        props: { goToWallet },
+      });
 
-      const articles = container.querySelectorAll("article");
+      const articles = queryAllByTestId("account-card");
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/NnsAccounts.spec.ts:72]: Use a more descriptive variable name than articles?

File: frontend/src/tests/lib/pages/NnsAccounts.spec.ts [hunk 2]
```diff
@@ -81,9 +82,11 @@ describe("NnsAccounts", () => {
         hardwareWallets: [mockHardwareWalletAccount],
         certified: true,
       });
-      const { container } = render(NnsAccounts, { props: { goToWallet } });
+      const { queryAllByTestId } = render(NnsAccounts, {
+        props: { goToWallet },
+      });
 
-      const articles = container.querySelectorAll("article");
+      const articles = queryAllByTestId("account-card");
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/NnsAccounts.spec.ts:89]: same

**Review by dskloetd — APPROVED** [review]

File: frontend/src/tests/lib/pages/NnsAccounts.spec.ts (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/NnsAccounts.spec.ts]: done

File: frontend/src/tests/lib/pages/NnsAccounts.spec.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/NnsAccounts.spec.ts]: Done!

**Pull request merged**